frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -0,0 +1,332 @@
+<template lang="pug">
+page-section.c-section(:title='L("Invite links")')
+  template(#cta='')
+    .select-wrapper.c-select-wrapper(:class='{ focus: ephemeral.selectbox.focused }')
+      select.button.is-small.is-outlined.c-select(
+        ref='select'
+        v-model='ephemeral.selectbox.selectedOption'
+        @change='unfocusSelect'
+      )
+        option(value='Active') {{ L('Active links') }}
+        option(value='All') {{ L('All links') }}
```

Comment:
Verify this on firefox, it doesn't look very pretty: 
<img width="144" alt="image" src="https://user-images.githubusercontent.com/14869087/67155455-a1279000-f318-11e9-9156-e1ae46c85e9b.png">

Reply:
This is a bug I haven't figured out how to fix that also exists in chrome. The way it's implemented currently (using Javascript)is the button loses focus when 'change' event occurs, which is not the case when we **re-select the option that is already selected**. Would you have any suggestion on fixing this problem?

Reply:
the issues was only the font, it's fixed as you did :)